test/python/gemm/frost/test_matmul.py

```
@@ -3403,3 +3404,233 @@ def test_sm120_render_smoke() -> None:
+
+
+@requires_sm100
+def test_splitk_more_slices_than_k_tiles():
+    # DEFAULT_CONFIG cta_tile_k = 64 bf16 elems -> K=128 has 2 tiles < S=4.
+    from cudnn.frost.workspace import Workspace
+
+    g, A, Bt, C = _splitk_graph(1, 256, 256, 128)
+    compiled = _jit_splitk(g, 4)
+    a, b, c, _ = _splitk_data(1, 256, 256, 128)
+    buf = torch.empty(compiled.workspace_bytes, dtype=torch.uint8, device="cuda")
+    with pytest.raises(ValueError, match="exceeds the .* K tile"):
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)